packages/admin/admin-theme/src/componentsTheme/MuiDataGrid.ts

```diff
@@ -0,0 +1,71 @@
+import type {} from "@mui/x-data-grid/themeAugmentation";
+
+import { mergeOverrideStyles } from "../utils/mergeOverrideStyles";
+import { GetMuiComponentTheme } from "./getComponentsTheme";
+
+export const getMuiDataGrid: GetMuiComponentTheme<"MuiDataGrid"> = (component) => ({
+    ...component,
+    styleOverrides: mergeOverrideStyles<"MuiDataGrid">(component?.styleOverrides, {
+        root: {
+            backgroundColor: "white",
+        },
+        columnHeader: {
+            "&:focus": {
+                outline: "none",
+            },
+            "&:focus-within": {
+                outline: "none",
+            },
+        },
+        columnHeadersInner: {
+            padding: "0 10px",
+        },
+        row: {
+            padding: "0 10px",
+        },
+        cell: {
+            borderTop: "1px solid #D9D9D9",
+            "&:focus": {
+                outline: "none",
+            },
+            "&:focus-within": {
+                outline: "none",
+            },
+        },
+        iconSeparator: {
+            backgroundColor: "#D9D9D9",
+            width: "2px",
+            height: "20px",
+            marginRight: "10px",
+        },
+        filterForm: {
+            padding: 20,
+            "& .MuiInputLabel-root": {
+                display: "none",
+            },
+            "& .MuiInput-root": {
+                marginTop: 0,
+            },
+            "& .MuiInputAdornment-root": {
+                paddingRight: 5,
+            },
+        },
+        filterFormDeleteIcon: {
+            justifyContent: "center",
+            "& .MuiSvgIcon-root": {
+                width: 16,
+                height: 16,
+            },
+        },
+        filterFormColumnInput: {
+            marginRight: 20,
+            marginLeft: 20,
+        },
+        filterFormOperatorInput: {
+            marginRight: 20,
+        },
+        paper: {
+            boxShadow: "0 0 8px 0 rgb(0 0 0 / 10%)",
```

Comment:
If you have access to the props of paper, you can set `elevation={1}` instead.

Otherwise you could use the theme-shadow from here: https://github.com/vivid-planet/comet/blob/main/packages/admin/admin-theme/src/shadows.ts#L5
But you'd have to add the shadows to `themeData` in `getComponentsTheme.ts`.

```suggestion
            boxShadow: shadows[1],
```

Reply:
You need to add the theme-shadows to `themeData` in `getComponentsTheme.ts` to use it here. 

Also you need to get the theme-values from the function-params to use them:
```diff
- export const getMuiDataGrid: GetMuiComponentTheme<"MuiDataGrid"> = (component) => ({
+ export const getMuiDataGrid: GetMuiComponentTheme<"MuiDataGrid"> = (component, { palette, shadows }) => ({
```